x-pack/plugins/actions/server/plugin.ts

```diff
@@ -285,7 +283,7 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
             savedObjectsClient: context.core.savedObjects.client,
             actionTypeRegistry: actionTypeRegistry!,
             defaultKibanaIndex,
-            scopedClusterClient: adminClient!.asScoped(request),
+            scopedClusterClient: context.core.elasticsearch.adminClient,
```

Comment:
Oops. `context.core.elasticsearch.adminClient` when accessing via `context`, but `leagacy.adminClient` in the elasticsearch start contract. Can be done in a separate PR. Up to you

Reply:
yeah, changing that API should be less work than the current changes, but I think it's best to isolate in a separate PR. I've added a task to https://github.com/elastic/kibana/issues/55397